src/Header.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import './tomato.jpg';
+import './Header.css';
+
+function Header() {
+  return (
+    <section className='header'>
+      <img className='tomato-image' src='./tomato.jpg' alt='basic tomato' />
```

Comment:
Need to figure out why our tomato won't load

Reply:
I've added an issue for this, a `bug` label is included.